server/routes/document.py

```
@@ -65,9 +112,11 @@ def post(self):
             )
             db.session.add(new_doc)
             db.session.commit()
-        except Exception:
+        except ValueError as e:
+            return error(str(e), 400)
+        except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)